index.html

```diff
@@ -70,51 +54,52 @@ <h1 class="headline-greeting">
                     <ul class="headline-socialmedia-icons-container">
                         <li>
                             <a href="#">
-                                <img src="images/Twitter icon.svg" alt="Twitter
-                                    icon" />
+                                <img src="images/Twitter icon.svg"
+                                    alt="Twitter icon" />
                             </a>
                         </li>
                         <li>
                             <a
-                                href="https://www.linkedin.com/in/noël-nomgne-foka-063013231/">
+                                href="https://www.linkedin.com/noelfoka/">
                                 <img src="images/Linkedin icon.svg"
-                                    alt="Linkedin icon" />
+                                    alt="LinkedIn icon" />
                             </a>
                         </li>
                         <li>
                             <a href="#">
-                                <img src="images/Medium icon.svg" alt="Medium
-                                    icon" />
+                                <img src="images/Medium icon.svg"
+                                    alt="Medium icon" />
                             </a>
                         </li>
                         <li>
                             <a href="https://github.com/noelfoka">
-                                <img src="images/Github icon.svg" alt="Github
-                                    icon" />
+                                <img src="images/Github icon.svg"
+                                    alt="Github Icon" />
                             </a>
                         </li>
                         <li>
                             <a href="#">
-                                <img src="images/hand icon.svg" alt="Hand icon" />
+                                <img src="images/hand icon.svg" alt="Hand
+                                    Icon" />
                             </a>
                         </li>
                     </ul>
                 </div>
             </div>
         </section>
         <!-- Works -->
-        <section class="works-container" id="portfolio">
+        <section class="works-container">
             <!-- Card -->
             <article class="card-container">
                 <!-- Card preview image container-->
                 <div class="card-preview-container">
                     <img
                         class="mobile"
                         src="images/Snapshoot Portfolio1.png"
-                        alt="Tonic sanpshoot" />
+                        alt="Tonic Sanpshoot" />
```

Comment:
- [ ] You're doing an amazing job with your cards! They're looking really good. I just wanted to offer a small suggestion to help **make them even better and more closely resemble the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=34-848&t=Wl4CVm5bcwVO9ujO-0)**. Would you mind checking the `width` of your cards? It seems like **they are slightly wider than in the given template.** Making this adjustment will bring them in line with the desired design. 👌

   ### Your design:
   ![image](https://github.com/noelfoka/portfolio-Microverse-version/assets/107952724/694e2f9c-75d2-41f1-81ad-0757b22eb223)

   ### Figma design:
   ![image](https://github.com/noelfoka/portfolio-Microverse-version/assets/107952724/80621d26-8e66-4d4b-a34e-5bbad96661f3)